spec/decorator_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../decorator'
+require_relative '../person'
+
+describe Nameable do
+  context 'Test for Nameable class' do
+    person = Person.new(22, name: 'maximilianus')
+    person.correct_name
+    capitalized_person = CapitalizeDecorator.new(person)
+    capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+
+    it 'checks if correcting of names is working' do
+      expect(capitalized_person.correct_name).to eql('Maximilianus')
+      expect(capitalized_trimmed_person.correct_name).to eql('Maximilian')
+    end
+  end
```

Comment:
- [ ] @Team Great work adding a test for the `Nameable` class, however, I didn't test for `Decorator`, `CapitalizeDecorator`, and `TrimmerDecorator` classes, kindly consider adding tests for these classes methods individually as you did for other classes eg person student.